jupyter/datascience/ubi8-python-3.8/Dockerfile

```diff
@@ -11,9 +11,14 @@ LABEL name="odh-notebook-jupyter-datascience-ubi8-python-3.8" \
     io.openshift.build.source-location="https://github.com/opendatahub-io/notebooks/tree/main/jupyter/datascience/ubi8-python-3.8" \
     io.openshift.build.image="quay.io/opendatahub/notebooks:jupyter-datascience-ubi8-python-3.8"
 
+WORKDIR /opt/app-root/bin
+
 # Install Python packages and Jupyterlab extensions from Pipfile.lock
 COPY Pipfile.lock ./
 
+# Copy Elyra setup to utils so that it's sourced at startup
+COPY setup-elyra.sh ./utils/
+
 RUN echo "Installing softwares and packages" && micropipenv install && rm -f ./Pipfile.lock
 
```

Comment:
It seems that the Python version displayed on the launcher is not inherited from minimal. 
Could you please add the following command on the data-science dockerfile? 

```
# Replace Notebook's launcher, "(ipykernel)" with Python's version 3.x
RUN sed -i -e "s/Python.*/$(python --version | awk '{print "Python", $2}' | cut -d'.' -f-2)\",/" /opt/app-root/share/jupyter/kernels/python3/kernel.json
```
Moreover, update the corresponding command into the minimal dockerfile, and remove it from the TrustyAi dockerfile, because it will be inherited by the datascience dockerfile. 

This cmd displays the python version without the patch version, so eventually, we could close also this issue: https://github.com/opendatahub-io/notebooks/issues/49

Reply:
Seems like this cmd is needed explicitly in each image build.
As the jupyter setup is getting overridden in build-time, the base image change of the kernel version is not carried over.
Tested on different images. 
Included it in another images as well.

Reply:
Sure, let's leave it for later on, to unblock this pr